examples/js/postprocessing/ManualMSAARenderPass.js

```diff
@@ -20,6 +20,12 @@ THREE.ManualMSAARenderPass = function ( scene, camera ) {
 	this.sampleLevel = 4; // specified as n, where the number of samples is 2^n, so sampleLevel = 4, is 2^4 samples, 16.
 	this.unbiased = true;
 
+	this.clearColor = clearColor;
+	this.clearAlpha = ( clearAlpha !== undefined ) ? clearAlpha : 1;
+
+	this.oldClearColor = new THREE.Color();
```

Comment:
I can do that.  I'll update RenderPass as well as I copied the pattern from that class and it thus has the same issues.

https://github.com/mrdoob/three.js/blob/master/examples/js/postprocessing/RenderPass.js#L17